packages/bff/src/services/selfcareService.ts

```diff
@@ -59,6 +80,46 @@ export function selfcareServiceBuilder(
         queries: { userIdForAuth: userId },
       });
     },
+
+    async getInstitutionUsers(
+      tenantId: TenantId,
+      userId: UserId | undefined,
+      query: string | undefined,
+      { authData, logger, headers }: WithLogger<BffAppContext>
+    ): Promise<bffApi.Users> {
+      logger.info(`Retrieving users for institutions ${tenantId}`);
+
+      const roles = authData.userRoles;
+      const requesterId = authData.organizationId;
+      if (!userId) {
+        throw missingUserId();
+      }
```

Comment:
I don't see this check in the Scala code, the userId (personId parameter from FE) can be optional and is passed down as optional to the selfcare client:
<img width="691" alt="image" src="https://github.com/user-attachments/assets/a913fbc7-4c28-462c-b374-aa3b2afd1ff3">

Reply:
my fault

Reply:
solved with [commit](https://github.com/pagopa/interop-be-monorepo/pull/986/commits/fcd8ac146a590bf51e60f0b27f54441690688066)